distribution/pom.xml

```diff
@@ -627,6 +627,8 @@
                                         <argument>io.confluent.druid.extensions:confluent-extensions</argument>
                                         <argument>-c</argument>
                                         <argument>org.apache.druid.extensions.contrib:opentelemetry-emitter</argument>
+                                        <argument>-c</argument>
+                                        <argument>org.apache.druid.extensions.contrib:druid-jose4-</argument>
```

Comment:
probably not a good idea to end the extension name with a dash. And the library we use to validate JWTs is not really that relevant. I think something like druid-jwt-extensions would be more specific